src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -121,6 +121,8 @@ private void testSubmitAction() throws Exception {
         submitPage = loginToStudentFeedbackSubmitPage("Alice", "Open Session");
```

Comment:
I beleive `InstructorFeedbackSubmitPageUiTest` should also be updated (adding test cases).

Reply:
Updated test cases :smile: